src/NUnitFramework/nunitlite.runner/AutoRun.cs

```diff
@@ -102,9 +102,9 @@ public int Execute(string[] args)
             }
 #endif
 
-            _textUI.DisplayTestFiles(new string[] { callingAssembly.GetName().Name });
             _textUI.DisplayRuntimeEnvironment();
-            _textUI.DisplayRequestedOptions();
+            _textUI.DisplayTestFiles(new string[] { callingAssembly.GetName().Name });
+//            _textUI.DisplayRequestedOptions();
```

Comment:
Since most of DisplayRequestedOptions is about filters, is the intention to remove this and allow the DisplayTestFilters to replace it? If so, we should remove it here and in Portable.